specification/billing/resource-manager/Microsoft.Billing/stable/2023-04-01/billingProfile.json

```diff
@@ -69,327 +82,540 @@
-          "description": "Billing address."
+          "description": "Address details.",
+          "allOf": [
+            {
+              "$ref": "./types.json#/definitions/AddressDetails"
+            }
+          ]
         },
         "currency": {
           "description": "The currency in which the charges for the billing profile are billed.",
-          "readOnly": true,
-          "type": "string"
+          "type": "string",
+          "readOnly": true
         },
         "displayName": {
-          "description": "The name of the billing profile.",
+          "description": "The name of the billing profile for the invoice section or customer.",
           "type": "string"
         },
         "enabledAzurePlans": {
           "description": "Information about the enabled azure plans.",
+          "type": "array",
           "items": {
             "$ref": "./types.json#/definitions/AzurePlan"
           },
-          "type": "array"
+          "x-ms-identifiers": []
         },
         "hasReadAccess": {
           "description": "Indicates whether user has read access to the billing profile.",
-          "readOnly": true,
-          "type": "boolean"
+          "type": "boolean",
+          "readOnly": true
         },
         "indirectRelationshipInfo": {
-          "$ref": "#/definitions/IndirectRelationshipInfo",
           "description": "Identifies the billing profile that is linked to another billing profile in indirect purchase motion.",
-          "readOnly": true
+          "allOf": [
+            {
+              "$ref": "#/definitions/IndirectRelationshipInfo"
+            }
+          ]
         },
         "invoiceDay": {
-          "description": "The day of the month when the invoice for the billing profile is generated.",
           "format": "int32",
-          "readOnly": true,
-          "type": "integer"
+          "description": "The day of the month when the invoice for the billing profile is generated.",
+          "type": "integer",
+          "readOnly": true
         },
         "invoiceEmailOptIn": {
           "description": "Flag controlling whether the invoices for the billing profile are sent through email.",
-          "type": "boolean"
+          "type": "boolean",
+          "readOnly": true
         },
-        "invoiceSections": {
-          "$ref": "#/definitions/InvoiceSectionsOnExpand",
-          "description": "The invoice sections associated to the billing profile. By default this is not populated, unless it's specified in $expand."
+        "invoiceRecipients": {
+          "description": "The list of email addresses to receive invoices by email for the billing profile.",
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "x-ms-identifiers": []
         },
         "poNumber": {
           "description": "The purchase order name that will appear on the invoices generated for the billing profile.",
```

Comment:
nit: "The default purchase order number that will appear on the invoices generated for the billing profile."